java/spacewalk-java.changes.rjpmestre.chained_container_proxies_configuration_refactor

```diff
@@ -0,0 +1,2 @@
+Chained proxy configuration create refactoring
```

Comment:
I think what you need here is:
```suggestion
- Chained proxy configuration create refactoring
```
And the second line is probably not needed...

Am I right @deneb-alpha / @raulillo82? Maybe I am getting a bit rusty :-D